### PR TITLE
feat(map): add click handler to open address in maps app

### PR DESCRIPTION
**Pull request by aemkei**

### CODE REVIEW

File: assets/scripts/views/infowindow.js [hunk 1]
```diff
@@ -43,6 +45,17 @@ export default class {
   }
 
   /**
+   * Open the default maps application.
+   */
+  openMaps() {
+    const position = this.hotspot.position,
+      query = this.hotspot.address + '/@' + `${position.lat},${position.lng}`,
```
Comment (ro-ka) [inline assets/scripts/views/infowindow.js]: Use the ES6 template stirng for the whole string.

File: assets/styles/partials/infowindow.styl [hunk 2]
```diff
@@ -16,6 +16,11 @@
     right: emCalc(15px)
     transform: translateY(-50%)
     z-index: 1
+  &__maplink
+    cursor: pointer
+    &:hover
+      color: $color-blue
+      text-decoration: underline
```
Comment (ro-ka) [inline assets/styles/partials/infowindow.styl]: Define this for links in general?

**Comment by ro-ka** [comment]
:checkered_flag:

File: assets/styles/partials/infowindow.styl (hunk 2 above)
Comment (aemkei) [inline assets/styles/partials/infowindow.styl]: Not sure if we'll have this style somewhere else.

Comment (ro-ka) [inline assets/styles/partials/infowindow.styl]: Okay.

**Pull request merged**